src/serac/physics/operators/odes.hpp

```diff
@@ -265,17 +270,6 @@ class FirstOrderODE : public mfem::TimeDependentOperator {
    */
   FirstOrderODE(int n, FirstOrderODE::State&& state, const EquationSolver& solver, const BoundaryConditionManager& bcs);
 
-  /**
```

Comment:
Was there a reason this was deleted?

Reply:
I think it was a vestigial prototype that I used before the `SecondOrderODE` PR. All `SecondOrderODEs` by default simultaneous implement both `mfem::TimeDependentOperator` and `mfem::SecondOrderTimeDependentOperator`, so the `SecondOrderODE` is a valid mfem object that can be used with solvers.